doc/installation.md

```diff
@@ -13,6 +13,7 @@ Installation of cameras on Linux is lengthy compared to other supported platform
 
 1. Ensure apt-get is up to date
   * `sudo apt-get update && sudo apt-get upgrade`
+  * **Note:** Or `sudo apt-get distr-upgrade` in case you have an older ubuntu 14.04 version (with deprecated nvidia-331), as this prevents the linux 4.4 kernel to compile properly.
```

Comment:
dist-upgrade not distr-upgrade

> Use `sudo apt-get distr-upgrade` instead of just `sudo apt-get upgrade` in case you have an older ubuntu 14.04 version (with deprecated nvidia-331* things installed), as this prevents the linux 4.4 kernel to compile properly.